docs/medical-api/api-reference/fhir/consolidated-data-query-get-single.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "Get Single Consolidated Data Query status"
+description: "Get the status of a specific query for patient's consolidated data."
+api: "GET /medical/v1/patient/{id}/consolidated/query/{requestId}"
+---
+
+This endpoint returns the status of the specified query.
+
+To trigger a new query you can
+[start a consolidated query](/medical-api/api-reference/fhir/consolidated-data-query-post). The result of the query will be sent through a [Webhook request](/medical-api/handling-data/webhooks#patient-consolidated-data)
+in FHIR-compliant format.
+
+## Path Params
+
+<ParamField path="id" type="string" required>
```

Comment:
I know the listing also has `id` for the patient ID, but I suggest we rename it to `patientId`, so its easier to populate the params on Mintlify's UI:

![image](https://github.com/user-attachments/assets/c7522ef1-2cc0-42e1-8abb-80096056ac3e)

Reply:
addressed